shared/editor/embeds/Abstract.test.ts

```diff
@@ -1,8 +1,8 @@
-import Abstract from "./Abstract";
+import { URL_REGEX } from "./Abstract";
```

Comment:
The embeds were problematic in `node@16.16.0`. It occurred that maybe this would fix it since we don't really require the entire component for tests. I tried running it multiple times and the leaks came to halt, although I didn't spot considerable difference in heap usage.

On the other hand, in `node@16.10.0` and `node@14.19.3`, these seemed to behave sanely with `--detect-leaks` flag.

Reply:
Oof, this is a painful change to make across all embeds though – and now there is an implicit requirement of both exports just for test compatibility. 

I think that the `/shared/` tests should just be split out into their own `test:shared` command that runs separately and if they're a touch leaky it at least won't affect the other server tests

Reply:
I'm going to split these out now

Reply:
Shouldn't have any conflicts, but heads up: https://github.com/outline/outline/pull/3951

Reply:
Lets remove these changes in embeds, otherwise you killed it – memory behavior is great now!

Reply:
Ahh..almost forgot! Thanks for pointing out. Reverted.